sourcegraph/about
# Code Intelligence Platform page

Closes #5608. Creates new page & carousel redesign option.

### Changelog
1. Creates new landing page
2. Adds new design option for the `CustomCarousel` ([ref thread](https://www.figma.com/file/s4z11CR4RuUfR37Q70Yi1W?node-id=42:5979#238612937)). Adds new interim prop `animateTransition` to trigger this design. If this design is approved we can remove the prop and apply the new styling to the rest of the `CustomCarousel`s.
3. New carousel design is: a fade in of the active item & indicator, indicator hover causes item change as well.
4. Makes carousels keyboard friendly (buttons were undetectable before).
5. Small adjustments to spacing & typography to files touched (to make them compliant with the DLS).

### Notes
- Focus on `CustomCarousel.tsx` & `code-intelligence-platform.tsx`

### Test
1. Ensure prettier has standardized the proposed changes.
2. Nav to `/code-intelligence-platform` and ensure design matches on all screen sizes
3. Ensure no breaking changes to other carousel instances

https://user-images.githubusercontent.com/59381432/182909281-b173f7b8-5b03-4fbc-97e6-32fe1001650a.mov
